src/WindowManager.cpp

```diff
@@ -107,10 +108,52 @@ void WindowManager::checkForUserChanges() {
         mWasPressed = false;
     }
 
-    if (isKeyPressed(GLFW_KEY_ESCAPE)) {
```

Comment:
Put in
```
int count;
const unsigned char* btns =  glfwGetJoystickButtons(GLFW_JOYSTICK_1, &count);
```

then in the if statement below edit it to be
`if (!menuKeyWasPressed && (isKeyPressed(GLFW_KEY_ESCAPE) || (btns[9] == GLFW_PRESS))`

that'll add gamepad support to get to the menu.